scripts/linters/python_linter.py

```diff
@@ -42,12 +42,33 @@
 # pylint: disable=wrong-import-order
 # pylint: disable=wrong-import-position
 from pylint import lint  # isort:skip
+from pylint.reporters import text# isort:skip
 import isort  # isort:skip
 import pycodestyle # isort:skip
 # pylint: enable=wrong-import-order
 # pylint: enable=wrong-import-position
 
 
+class PylintReport(python_utils.OBJECT):
```

Comment:
I actually don't really have any idea what a "pylint report" object is. Can you maybe give a bit more explanation about what it is, what it does, what it's used for, etc.?

Reply:
Looking at where this is used -- this is just a report stream, right? If so, you can call it something generic like StringMessageStream. The variable name can still be pylint_message_stream or similar.